make 404-page better than ever before

kahlstrm opened the pull request:
Improve 404 page not found page with bad localization support and a link to homepage.

Relies on typography from #48.

Closes #37.

kahlstrm committed:
make 404-page better than ever before

tuukka-miettinen reviewed (approved):
Could also create an issue for the nearest-matching path from strapi link in 404 page

PurkkaKoodari commented on src/pages/404.tsx:
```diff
@@ -1,13 +1,36 @@
+        <SideBar>
+          {locale === "fi" ? (
+            <>
+              <h1>Sivua {pathname} ei löytynyt</h1>
```
We should change this to `<H1>` from typography.tsx when merging (or rebase/merge it to be so beforehand)

kahlstrm committed:
fix h1 to typography.tsx H1

Otju committed:
Added DinoGame to 404 Page. Changed 404 page background to white. Added VT323 pixel font. Ran npm audit fix.

Otju committed:
Added start wipe effect to DinoGame. Removed border from DinoGame. Fixed jump at restart.

Otju committed:
Fixed error message

PurkkaKoodari committed:
Add widths to SVGs to fix .width on Firefox

PurkkaKoodari committed:
Preload DinoGame images and clean up type casts

PurkkaKoodari committed:
Rewrite most of DinoGame

- Stop rendering when game is not running
- Use time instead of frames to work regardless of refresh rate
- Wait for images to load before rendering
- Refactored code to be much clearer and added comments
- Some other bugfixes

PurkkaKoodari committed:
Don't render empty sidebar

PurkkaKoodari committed:
Comment awaitProperties

kahlstrm reviewed (commented):
Very good very nice

PurkkaKoodari reviewed (approved):
"Very good very nice" --@kqlski

PurkkaKoodari commented:
Minor issue: the 404.html page is rendered with its in-filesystem pathname, which flashes before the page hydrates. Maybe this would be better without the pathname, then.

![image](https://user-images.githubusercontent.com/8439661/210569860-34c71688-e5ae-406b-a187-0d3be6a3dbea.png)
